certified-connectors/Redque/apiDefinition.swagger.json

```diff
@@ -1,4 +1,3551 @@
 {
+  "swagger": "2.0",
+  "basePath": "/",
+  "consumes": [],
+  "host": "demoapi.redque.com",
```

Comment:
@jirisodomka can you please confirm that this is the correct host URL? can you please confirm that it is not in testing or staging mode?